### PR TITLE
20151030 deployer fixes

### PR DESCRIPTION
**Pull request by sosiouxme**
Fix bug 1276235

### CODE REVIEW

**Comment by sosiouxme** [comment]
@ewolinetz @jcantrill PTAL

File: deployment/templates/es.yaml [hunk 1]
```diff
@@ -42,12 +42,7 @@ objects:
             kind: ImageStreamTag
             name: logging-elasticsearch:latest
       strategy:
-        resources: {}
-        rollingParams:
-          intervalSeconds: 1
-          timeoutSeconds: 600
-          updatePeriodSeconds: 1
-        type: Rolling
+        type: Recreate
```
Comment (ewolinetz) [inline deployment/templates/es.yaml]: I forget, does this also need recreate params as well?

Comment (sosiouxme) [inline deployment/templates/es.yaml]: There aren't any needed for recreate (it's pretty simple).
https://docs.openshift.com/enterprise/3.0/dev_guide/deployments.html#recreate-strategy

**Comment by ewolinetz** [comment]
LGTM

**Pull request merged**